pages/about.html

```diff
@@ -1,12 +1,73 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Sana</title>
+    <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+    <header>  
+        <h1>A Little About ME!</h1>
+        <div class="top">
+            <a href="index.html"> Swiftie Website</a>
+            <a href="portfolio.html"> My Portfolio</a>
+        </div>
+    </header>
+
+<section>
+    <div class="intro">
+        
+        <img alt="A picture of me on a kayak laughing" src="../assets/my_photo.png"/>
+        
+        <div>
+            <h1> Hi I am Sana!</h1> 
+            <ul>
+                <li>
+                    <p>
+                    I am a recent Biomedical Engineering graduate from Georgia Tech. I enjoy painting, practicing jujitsu, playing piano, watching soccer and TED talks. I am constantly doing something, whether it's working on my research or building things.
+                    </p>
+                </li>
+
+                <li>
+                    <p>
+                        I am an adventurous person and I love to visit new places and do new activities and meet new people. Before I die, I want to have managed to travel the world. My favorite quote is by Wayne Gretzky "You miss 100% of the shots you don't take".
+                    </p>
+                </li> 
+            </ul>
+        </div>
+    </div>
+</section>
+
+<section class="contact">
+    <nav>
+        <h1> Contact ME! </h1>
+        <a href="https://www.facebook.com/sana.pournaghshband.22/">
+            <img class="logo" alt="Fb Logo" src="../assets/fb-logo.png" />
+        </a>
+
+        <a href="https://twitter.com/sana_p22">
+            <img class="logo" alt="Twitter Logo" src="../assets/tw.png"/>
+        </a>
+
+        <a href="https://sanakakadan22.wixsite.com/website">
+            <img class="logo" alt="My Website Logo" src="../assets/web.png"/>
+        </a>
+
+        <a href="https://www.linkedin.com/in/sanapournaghshband/">
+            <img class="logo" alt="LinkedIN Logo" src="../assets/linkedin.png"/>
+        </a>
+
+        <a href="https://github.com/sanakakadan22">
+            <img class="logo" alt="Github Logo" src="../assets/github.png"/>
+        </a>
+    </nav>
+</section>
```

Comment:
Putting the content on lines 45 to 64 inside the nav element is great. Since it is inside a nav element, you do not need to put nav inside section. You can put the class "contact" on the nav element instead. 

I would say that your nav element can be moved into the footer since it typically contains information about the author of the section, copyright data or links to related documents.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer